pyproject.toml

```
@@ -24,7 +24,7 @@ dependencies = [
     "aiohttp",
     "PyYAML",
     "click",
-    "fastapi[all]",
+    "fastapi>=0.112.0",
```

```suggestion
    "fastapi[all]>=0.112.0",
```

unless there was a refactor of the dependencies? I don't know what we required from [all]

---

I think we should remove the fastapi[all] as it is causing problems on main as well...I have created a PR to address that issue https://github.com/DiamondLightSource/blueapi/pull/607